codelabs/tracing-ea/index.html

```diff
@@ -9,7 +9,7 @@
   <title>Distributed Tracing EA</title>
   <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400|Roboto:400,300,400italic,500,700|Roboto+Mono">
   <link rel="stylesheet" href="//fonts.googleapis.com/icon?family=Material+Icons">
-  <link rel="stylesheet" href="https://storage.googleapis.com/codelab-elements-tmp/codelab-elements.css">
+  <link rel="stylesheet" href="https://storage.googleapis.com/codelab-elements/codelab-elements.css">
```

Comment:
hey @pl89 - we currently have an outstanding bug in the codelabs css generation and we need to change this file manually, so can you add the -tmp back in. and that’s about it!

Reply:
Commit the change requested, thank you!